lib/biscuit/execution_error.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Biscuit
+  class ExecutionError < StandardError
+    def initialize(stdout, stderr)
```

Comment:
I'd feel a lot better about this if you changed the `stdout` parameter to be optional - I've run into a lot of places in the past where it was really irritating that someone had changed the initialization signature of a subclass.